agnpy/absorption/absorption.py

```diff
@@ -443,6 +445,113 @@ def evaluate_tau_blr_mu_s(
         )
         return (prefactor * integral).to_value("")
 
+    @staticmethod
+    def compute_integrand(cubepy_params_array, R_line, mu_s, epsilon_1, epsilon_line):
```

Comment:
since the function is only for BLR and it is in general Absorption class it would be good to have 'blr' in the name

Reply:
renamed to `compute_integrand_for_blr`